### PR TITLE
DOC/TST: doc added to travis, also test for py36

### PR DESCRIPTION
**Pull request by licode**

### CODE REVIEW

**Comment by tacaswell** [comment]
Is it looking at the correct conda channels?

**Comment by tacaswell** [comment]
ping @danielballan for how to make this auto-push

**Comment by danielballan** [comment]
@licode 

1. Make a GH deploy key for this repo.
2. Encrypt it using travis's ruby CLI and commit the result (an `.enc` file) to version control.
3. Copy/paste from bluesky, ophyd, databroker, or amostra.

**Comment by licode** [comment]
ready to go, @danielballan ?

**Comment by danielballan** [comment]
Close! The docs build with two warnings. We have lately tried to get all the docs building with 0 warnings. Can these be fixed in the docs as part of this PR? The first error is [here](https://travis-ci.org/NSLS-II/suitcase/jobs/225262823#L865); the second one is just below it, also in red.

**Comment by licode** [comment]
good catch. I will fix those.

File: doc/source/index.rst [hunk 1]
```diff
@@ -30,20 +30,20 @@ in this function to define specifically which data sets you want to output.
 
     from suitcase import hdf5
     hdr = db[123]
-    un_wanted_fields = [‘A’, ‘B’, ‘C’]
+    un_wanted_fields = ['A', 'B', 'C']
```
Comment (danielballan) [inline doc/source/index.rst]: I thought those quotes looked funny. I should have spoken up.

**Comment by danielballan** [comment]
Looks good Li. If Travis builds clean (look for "built with 0 warnings") then self-merge.

File: doc/source/index.rst (hunk 1 above)
Comment (licode) [inline doc/source/index.rst]: quotation mark in Chinese

**Comment by licode** [comment]
finally there is no warning.

**Pull request merged**